md/ts-jsdoc.md

```diff
@@ -124,6 +155,16 @@ export type IntersectionType = Type1 & Type2
 /** @type { import('./types').IntersectionType } */
 const list
 ```
+You can also organise your source types in the same way as [vendored types](#vendor-type-declarations). 
```

Comment:
I think leaning into "vendored types" would be a mistake & I would really like for it to be stop-gap solution until it becomes obsolete, which is why I find introduction `self` pattern to be a bad idea.

Can we instead just have some directory inside src and import by relative paths ?

Reply:
whats the difference ?
we already use `types.ts` files just because the ts compiler outputs those to the dist folder and `.d.ts` aren't outputed. `.d.ts` allows stuff that `.ts` doesn't, paths are simpler, tooling just needs to copy the types folder to dist at build time (any other solutions with `.d.ts` files is more complex) and in a monorepo we can even skip creating a new package just for types.